packages/circuits/circuits/preventDoubleAction.circom

```diff
@@ -0,0 +1,94 @@
+pragma circom 2.0.0;
+
+include "./circomlib/circuits/poseidon.circom";
+
+template preventDoubleAction(STATE_TREE_DEPTH, EPOCH_KEY_NONCE_PER_EPOCH) {
```

Comment:
The name of the template should be capitalized because it's capitalized in `circuits.mjs`. So `PreventDoubleAction`